RELEASES.md

```diff
@@ -1,3 +1,135 @@
+Version 1.85.0 (2025-02-20)
+==========================
+
+<a id="1.85.0-Language"></a>
+
+Language
+--------
+- [Support var-args for more calling conventions](https://github.com/rust-lang/rust/pull/116161) 
+  This stabilizes usage of `...` variable args in extern declarations for `cdecl`, `system`, `aapcs`, `sysv64`, `win64`, and `efiapi` calling conventions.
+- [Add `unpredictable_function_pointer_comparisons` lint to warn against function pointer comparisons](https://github.com/rust-lang/rust/pull/118833)
+- [Lint on combining `#[no_mangle]` and `#[export_name]` attributes.](https://github.com/rust-lang/rust/pull/131558)
+- [Stabilize `#[diagnostic::do_not_recommend]`](https://github.com/rust-lang/rust/pull/132056)
+- [Stabilize async closures (RFC 3668)](https://github.com/rust-lang/rust/pull/132706)
+- [The 2024 Edition is now stable.](https://github.com/rust-lang/rust/pull/133349)
+  See [the edition guide](https://doc.rust-lang.org/nightly/edition-guide/rust-2024/index.html) for more details.
+
+
+<a id="1.85.0-Compiler"></a>
+
+Compiler
+--------
+- [The unstable flag `-Zpolymorphize` has been removed](https://github.com/rust-lang/rust/pull/133883), see https://github.com/rust-lang/compiler-team/issues/810 for some background.
+- [Enable LSX feature for `loongarch64_unknown_linux_ohos`.](https://github.com/rust-lang/rust/pull/134751)
```

Comment:
Suggestion: probably remove this (maybe by closing the relnotes issue?), I know I was the one who tagged that with relnotes, but I think for Tier 3 targets we usually don't list enabling a target feature?